arrays/victor/C#_soft-188/3_task/3_task/3_task/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace _3_task
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] mas = { 1, 2, 5, 78, 2, 34, 248, 11, 9, 29, 0, 270 };
+            Console.WriteLine("Масив = 1, 2, 5, 78, 2, 34, 248, 11, 9, 29, 0, 270");
+            Array.Sort(mas);
+            int maxValue = mas[mas.Length - 1];
```

Comment:
In this task you can't use C# methods. Try to find the max element without using Sort() method.